wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/ConflictExceptionError.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git;
+
+import org.eclipse.che.dto.shared.DTO;
+
+import java.util.List;
+
+/**
+ * Created by I053322 on 7/10/2016.
+ */
```

Comment:
Please make this javadoc in common way like this:
`/**`
 `* @author <Name> <Surname> `
 `*/`
and also in other classes